pkg/server/register.go

```diff
@@ -296,7 +296,22 @@ func (i *InventoryServer) serveLoop(conn *websocket.Conn, inventory *elm.Machine
 			if err = json.Unmarshal(data, &smbiosData); err != nil {
 				return msgType, fmt.Errorf("cannot extract SMBios data: %w", err)
 			}
-			inventory.Name = buildName(smbiosData, inventory.Name)
+			inventory.Name = buildStringFromSmbiosData(smbiosData, inventory.Name)
+			logrus.Debug("Adding extra labels")
+			// Add extra label info from data coming from smbios
+			newLabels := map[string]string{
+				"manufacturer":    buildStringFromSmbiosData(smbiosData, "${System Information/Manufacturer}"),
+				"productName":     buildStringFromSmbiosData(smbiosData, "${System Information/Product Name}"),
+				"serialNumber":    buildStringFromSmbiosData(smbiosData, "${System Information/Serial Number}"),
+				"uuid":            buildStringFromSmbiosData(smbiosData, "${System Information/UUID}"),
+				"cpuManufacturer": buildStringFromSmbiosData(smbiosData, "${Processor Information/Manufacturer}"),
+				"cpuFamily":       buildStringFromSmbiosData(smbiosData, "${Processor Information/Family}"),
+				"cpuCore":         buildStringFromSmbiosData(smbiosData, "${Processor Information/Core Count}"),
+				"cpuThreads":      buildStringFromSmbiosData(smbiosData, "${Processor Information/Thread Count}"),
+			}
+			for k, v := range newLabels {
+				inventory.Labels[k] = strings.TrimSuffix(strings.TrimPrefix(v, "-"), "-")
```

Comment:
Do we need to trim `-`? isn't that "sanitized" in  buildStringFromSmbiosData() (so I would rather expect `_`)
We probably really want to review the buildStringFromSmbiosData ... I will give a look while at #145 😓